src/app/new-page/component/training-tip/training-tip.component.html

```diff
@@ -0,0 +1,2 @@
+<ng-container *ngIf="ready && (visible$ | async)"></ng-container>
+<!-- Training tip content goes here -->
```

Comment:
as it stands right now, there will never be any training tip rendering happening on knowgod.com. So you can probably get rid of the `<ng-container />` here

Reply:
Gotcha, if that's the case, we can remove the entire component.